xml/admin_rbd.xml

```diff
@@ -1101,6 +1290,47 @@ data blocks changed from 2097152 to 2560000</screen>
     </para>
    </tip>
   </sect2>
+  <sect2 xml:id="rbd-mirror-daemon">
+    <title>rbd-mirror Daemon</title>
+    <para>
+      The two <command>rbd-mirror</command> daemons are responsible for watching image journals
+      on the <literal>remote</literal> peer cluster and replaying the journal events against
+      the <literal>local</literal> cluster. The RBD image journaling feature records all
+      modifications to the image in the order they occur. This ensures that
+      a crash-consistent mirror of the remote image is available locally.
+    </para>
+    <para>
+      The <command>rbd-mirror</command> daemon is available within the
+      optional <command>rbd-mirror</command> distribution package.
```

Comment:
I would remove this comment. In SES7 we use docker images and no need to install additional package.